modules/dr-restoring-cluster-state.adoc

```diff
@@ -712,4 +716,21 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 
 To ensure that all workloads return to normal operation following a recovery procedure, restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, Operators, and third-party components.
 
-Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+[NOTE]
+====
+On completion of the previous procedural steps, you might need to wait a few minutes for all services to return to their restored state. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+Consider using the `system:admin` `kubeconfig` file for immediate authentication, because this method basis its authentication on SSL/TLS client certificates as against OAuth tokens. You can authenticate with this file by issuing the following command:
```

Comment:
The SSG states to use caution with "because" I think it is used correctly here, but you could just start a new sentence:
```suggestion
Consider using the `system:admin` `kubeconfig` file for immediate authentication. This method basis authentication on SSL/TLS client certificates as against OAuth tokens. You can authenticate with this file by issuing the following command:
```

Reply:
Yes, good point. Updated.